python/sglang/srt/models/qwen2_vl.py

```diff
@@ -605,7 +605,11 @@ def forward(
             ]
 
         positions = forward_batch.mrope_positions
-        if image_inputs is None or len(image_inputs) == 0:
+        if (
+            forward_batch.forward_mode.is_decode()
+            or image_inputs is None
```

Comment:
why this change?

Reply:
Llama vision requires passing image_inputs in both decode and prefill. However, qwen2 vl does not pass image_inputs when decoding, and it uses this as a judge to process the images or not...